apps/sparo-lib/src/services/GitSparseCheckoutService.ts

```diff
@@ -284,7 +289,13 @@ export class GitSparseCheckoutService {
   }
 
   private _getSkeletonPaths(): string[] {
-    const basicFolders: string[] = ['.vscode', 'common', 'common/sparse-profiles', 'scripts', 'plugins'];
+    const basicFolders: string[] = [
+      '.vscode',
+      'common',
+      'common/sparse-profiles',
+      'scripts',
+      'plugins'
```

Comment:
Oh I see, it is an internal API.  That's okay for this PR, but maybe it's time to consider introducing a Sparo config file?

Reply:
I would like to talk a little bit about this here. The first idea is to create a sparo configuration file under home folder.


1. Since JS callback function is need, it will be a JS file

**$HOME_FOLDER/.config/sparo/sparo.config.js**

```js
module.exports = {
  collectTelemetryAsync: function() {},
  callerPackageJson: { name: 'xxx', version: 'yyy' },
  additionalSkeletonFolders: ['foo', 'bar'],
}
```

2. The JS callback needs to import third party modules, which involves a dynamically installation or further global setup.

For example,

**$HOME_FOLDER/.config/sparo/sparo.config.js**

```
const { PackageJsonLookup } = require('@rushstack/node-core-library');

const callerPath = module.parent;

module.exports = {
  callerPackageJson: PackageJsonLookup.instance.tryGetPackageJsonFilePathFor(callerPath),
}
```

Things unclear:
1. where to get `@rushstack/node-core-library`
2. it depends on legacy API such as `module.parent`